src/EnergyPlus/VariableSpeedCoils.cc

```diff
@@ -5827,28 +5830,30 @@ namespace VariableSpeedCoils {
                 DefrostControl = StandardRatings::HPdefrostControl::Timed;
                 break;
             }
-            StandardRatings::CalcDXCoilStandardRating(state,
-                                                      varSpeedCoil.Name,
-                                                      varSpeedCoil.VarSpeedCoilType,
-                                                      varSpeedCoil.VSCoilType,
-                                                      varSpeedCoil.NumOfSpeeds,
-                                                      varSpeedCoil.MSRatedTotCap,
-                                                      varSpeedCoil.MSRatedCOP,
-                                                      varSpeedCoil.MSCCapAirFFlow,
-                                                      varSpeedCoil.MSCCapFTemp,
-                                                      varSpeedCoil.MSEIRAirFFlow,
-                                                      varSpeedCoil.MSEIRFTemp,
-                                                      varSpeedCoil.PLFFPLR,
-                                                      varSpeedCoil.MSRatedAirVolFlowRate,
-                                                      varSpeedCoil.MSRatedEvaporatorFanPowerPerVolumeFlowRate2017,
-                                                      varSpeedCoil.MSRatedEvaporatorFanPowerPerVolumeFlowRate2023,
-                                                      CondenserType,
-                                                      0, // varSpeedCoil.RegionNum, // ??
-                                                      varSpeedCoil.MinOATCompressor,
-                                                      varSpeedCoil.OATempCompressorOn,
-                                                      false, // varSpeedCoil.OATempCompressorOnOffBlank, // ??
-                                                      DefrostControl,
-                                                      ObjexxFCL::Optional_bool_const());
+            if (varSpeedCoil.RatedCapCoolTotal > 0.0 || varSpeedCoil.RatedCapCoolTotal == AutoSize) {
```

Comment:
Why is this needed here? Is there a divide by 0 in CalcDXCoilStandardRating? This may be another place where the developer did not anticipate a 0 rated capacity (i.e., lot's of coil models call this function). And isn't this called after sizing? if so then why is == AutoSize valid here? Protecting this here means that standard rating would not be reported to the eio, which is not a huge issue when capacity = 0.

Reply:
@rraustad I know that I was getting a crash here where this trap was not put in place.  So, I think this is a case of the developer not anticipating a 0 rated capacity.  Now, I did just try this without the AutoSize part and it still works.  So, like the IF at line 4001, I took out the AutoSize part and am running the test suite to see if that causes any issues.  If not, I will make another commit here shortly.